ect/ect/embed_graph.py

```diff
@@ -53,6 +61,50 @@ def add_nodes_from(self, nodes, coordinates):
         super().add_nodes_from(nodes)
         self.coordinates.update(coordinates)
 
+    def next_vert_name(self, s, num_verts=1):
```

Comment:
can we separate into type checking then increment integers or running a sub function `increment_string` for readability. took me a minute to figure out this part haha

Reply:
I separated this one off into another issue. I'm sure there's a more intelligent way to do this than what I was doing but I'm not ready to poke that bear today